hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -344,14 +344,27 @@ private StoreContext initializeStoreContext(ColumnFamilyDescriptor family) throw
   }
 
   private InetSocketAddress[] getFavoredNodes() {
-    InetSocketAddress[] favoredNodes = null;
```

Comment:
Does the code still compile after you remove this line? This is a behavior change? We have a favoredNodes class field in HStore?